### PR TITLE
Paper/vpn

### PR DESCRIPTION
**Pull request by czotti**

### CODE REVIEW

File: deep-learning/_posts/2017-08-03-video_propagation_networks.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  venue:   "CVPR 2017"
+---
+
+Paper :
+   - [project page](https://varunjampani.github.io/vpn/)
+   - [code](https://github.com/varunjampani/video_prop_networks)
+   - [suplementary material](https://varunjampani.github.io/papers/jampani17_VPN_supp.pdf)
+
+
+The goal of this papers is to take a video and a task image as input and apply the task image on the entire video, this task could be pixel segmentation, pixel colorization, etc.
```
Comment (jodoin) [inline deep-learning/_posts/2017-08-03-video_propagation_networks.md]: paper
entire video. This...

File: deep-learning/_posts/2017-08-03-video_propagation_networks.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<div align="middle">
+<img src="/deep-learning/images/vpn/architecture.png"/>
+</div>
+
+This model is used in a staged flavor by first generating a segmentation and refining it recursively.
+
+## Evaluation
+
+The VPN model is tested on three different propagation task, foreground mask, semantic label and color propagation.
```
Comment (jodoin) [inline deep-learning/_posts/2017-08-03-video_propagation_networks.md]: tasks :

Comment (jodoin) [reply deep-learning/_posts/2017-08-03-video_propagation_networks.md]: labeling

File: deep-learning/_posts/2017-08-03-video_propagation_networks.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+They use the [DAVIS](http://davischallenge.org/) dataset to evaluate their model. The tables show three metrics for the evaluation, Intersection over Union (IoU), contour accuracy ($$\mathcal{F}$$) and the temporal instability score ($$\mathcal{T}$$).
+
+- The results on Video Object Segmention of DAVIS:
+<div align="middle">
+<img src="/deep-learning/images/vpn/results_davis.png"/>
+</div>
+
+Their model with pretraining achieves an improvement of 5% on all the metrics and 10 times faster against the best reported method (OFL).
```
Comment (jodoin) [inline deep-learning/_posts/2017-08-03-video_propagation_networks.md]: while being 10 times faster than the ...

File: deep-learning/_posts/2017-08-03-video_propagation_networks.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The goal of this papers is to take a video and a task image as input and apply the task image on the entire video, this task could be pixel segmentation, pixel colorization, etc.
+
+## Models
+
+To achieve this they use two kinds of network, a [Bilateral network](https://arxiv.org/pdf/1412.6618.pdf){:target="_blank"}, and a Spatial network.
+
+The bilinear features used are defined by a 6 dimensional feature vector $$ F^i = (x, y, Y, Cb, Cr, t)^T $$. Where $$x$$ and $$y$$ are the pixel position in the image, $$Y$$, $$Cb$$, and $$Cr$$ the luma, blue-difference and red-difference chroma components, and $$t$$ the time of the frame in the video.
+
+The bilateral network performs video-adaptive spatiotemporal dense filtering that is given to the spatial network (regular CNN) used to produce the new segmentation from the previous frame segmentation as shown in Fig.3.
```
Comment (jodoin) [inline deep-learning/_posts/2017-08-03-video_propagation_networks.md]: what is guidance in Fig.3?

Comment (czotti) [reply deep-learning/_posts/2017-08-03-video_propagation_networks.md]: The Fig.3 is there to show the architecture and the connection between the bilateral network and the spatial network as well as the input of the bilateral network.

**Comment by czotti** [comment]
Can I merge it @jodoin ?

**Pull request merged**